symplyphysics/laws/conservation/mass_after_equals_to_mass_before.py

```diff
@@ -0,0 +1,53 @@
+from sympy import (Eq, solve, dsolve)
+from symplyphysics import (units, Quantity, Symbol, print_expression, Function, validate_input,
+    validate_output)
+from symplyphysics.core.expr_comparisons import expr_equals
+from symplyphysics.laws.conservation import mass_is_constant
+
+# Description
+## The total mass of the closed system is preserved.
+## For more information, see [mass_is_constant](./mass_is_constant.py ).
+
+# Law: m(t1) = m(t0)
```

Comment:
Now when you have this law, you can check if any example can be simplified by removing extra `dsolve()`